product/toolbars/container_image_center_tb.yaml

```diff
@@ -11,17 +11,11 @@
     :title: Configuration
     :text: Configuration
     :items:
-    - :button: container_image_edit
```

Comment:
@oschreib these features should be deleted from product features yaml as they have been removed from toolbar yaml.